webservice/routes/api.php

```diff
@@ -3,11 +3,13 @@
 Route::group([
     'middleware' => ['cors', 'api'],
 ], function () {
-    Route::post('/login', 'LoginController@login');
+    Route::post('/auth/issue', 'AuthController@issueToken');
```

Comment:
I'm OK with that. Because in the realm of **auth**, we have an **artifact** (token) and then we **do something** with it: Issue or Revoke.